src/components/data.js

```diff
@@ -0,0 +1,31 @@
+import {BsPeopleFill} from "react-icons/bs"
+import {BsFillKeyboardFill} from "react-icons/bs"
+import {BsFillFileEarmarkCodeFill} from "react-icons/bs"
+import {BsFillEmojiSmileFill} from "react-icons/bs"
+
+export const FeatureList = [
+  {
+    id: 1,
```

Comment:
ids might not have to be added here but can be rendered as you're iterating through the loop?